website/oauth/models/__init__.py

```diff
@@ -443,3 +443,63 @@ def absolute_api_v2_url(self):
     # used by django and DRF
     def get_absolute_url(self):
         return self.absolute_api_v2_url
+
+class ApiOAuth2PersonalToken(StoredObject):
+    """
+    Store information about recognized OAuth2 scopes. Only scopes registered under this database model can
+        be requested by third parties.
+    """
+    _id = fields.StringField(primary=True,
+                             default=lambda: str(ObjectId()))
+
+    token_id = fields.StringField(default=lambda: uuid.uuid4().hex,  # Not *guaranteed* unique, but very unlikely
+                               unique=True)
+
+    owner = fields.ForeignField('User',
+                                backref='created',
+                                index=True,
+                                required=True)
+
+    user_id = fields.StringField(default=owner)
```

Comment:
The `user_id` and `owner` fields appear to be redundant.

As far as I can tell, the duplication [is a CAS requirement](https://github.com/abought/cas-overlay/blob/develop/cas-server-support-osf/src/main/java/io/cos/cas/adaptors/mongodb/OpenScienceFrameworkPersonalAccessTokenHandler.java#L55-55). Is that correct?

If so, I recommend standardizing on only one field (and not maintaining two separate fields that do the same thing, but are each checked in different places). In this case `owner` is more consistent with the `ApiOauth2Application` collection; this may require coordinating a CAS change with Michael.